subsys/lorawan/services/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_sources_ifdef(CONFIG_LORAWAN_SERVICES
+        clock_sync.c
```

Comment:
clock_sync.c should probably be under `_ifdef(CONFIG_LORAWAN_APP_CLOCK_SYNC`

Reply:
True. Fixed.